client/src/pages/Signup.jsx

```diff
@@ -1,13 +1,286 @@
+		}
+	},
+	welcome: {
+		fontSize: 26,
+		paddingBottom: 20,
+		color: '#000000',
+		fontWeight: 500
+	},
+	heroText: {
+		fontSize: 26,
+		textAlign: 'center',
+		color: 'white',
+		marginTop: 30,
+		maxWidth: 300
+	},
+	overlay: {
+		backgroundImage: 'linear-gradient(180deg, rgb(58,141,255, 0.75) 0%, rgb(134,185,255, 0.75) 100%)',
+		backgroundSize: 'cover',
+		backgroundPosition: 'center',
+		flexDirection: 'column',
+		minHeight: '100vh',
+		paddingBottom: 145,
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center'
+	},
+	buttonHeader: {
+		display: 'flex',
+		alignItems: 'flex-start',
+		justifyContent: 'space-between',
+		flexDirection: 'column',
+		bgcolor: 'background.paper',
+		minHeight: '100vh',
+		paddingTop: 23
+	},
+	accBtn: {
+		width: 170,
+		height: 54,
+		borderRadius: 5,
+		filter: 'drop-shadow(0px 2px 6px rgba(74,106,149,0.2))',
+		backgroundColor: '#ffffff',
+		color: '#3a8dff',
+		boxShadow: 'none',
+		marginRight: 35
+	},
+	noAccBtn: {
+		fontSize: 14,
+		color: '#b0b0b0',
+		fontWeight: 400,
+		textAlign: 'center',
+		marginRight: 21,
+		whiteSpace: 'nowrap'
+	},
+	image: {
+		backgroundImage: 'url(./images/bg-img.png)',
+		backgroundRepeat: 'no-repeat',
+		backgroundSize: 'cover',
+		backgroundPosition: 'center'
+	},
+	box: {
+		padding: 24,
+		display: 'flex',
+		justifyContent: 'center',
+		alignItems: 'center',
+		minHeight: '100vh',
+		flexDirection: 'column',
+		maxWidth: 900,
+		margin: 'auto'
+	},
+	form: {
+		width: '100%', // Fix IE 11 issue.
+		marginTop: theme.spacing(1)
+	},
+	label: { fontSize: 19, color: 'rgb(0,0,0,0.4)', paddingLeft: '5px' },
+	submit: {
+		margin: theme.spacing(3, 2, 2),
+		padding: 10,
+		width: 160,
+		height: 56,
+		borderRadius: 3,
+		marginTop: 49,
+		fontSize: 16,
+		fontWeight: 'bold'
+	},
+	inputs: {
+		marginTop: '.8rem',
+		height: '2rem',
+		padding: '5px'
+	},
+	link: { textDecoration: 'none', display: 'flex', flexWrap: 'nowrap' }
+}));
+
+function useRegister() {
+	const login = (email, password) => {
+		console.log(email, password);
+	};
+	return login;
+}
+
+export default function Register() {
+	const classes = useStyles();
+	const [ open, setOpen ] = React.useState(true);
+
+	const register = useRegister();
+
+	const handleClose = (event, reason) => {
+		if (reason === 'clickaway') return;
+		setOpen(false);
+	};
 
-export default function Signup({ handleClick }) {
 	return (
-		<div>
-			<Typography>
-				<Link onClick={() => handleClick('login')}>Already have an account?</Link>
-			</Typography>
-		</div>
+		<Grid container component="main" className={classes.root}>
+			<CssBaseline />
+			<Grid item xs={false} sm={4} md={5} className={classes.image}>
+				<Box className={classes.overlay}>
+					<Hidden xsDown>
+						<img width={67} src="/images/chatBubble.png" />
+						<Hidden smDown>
+							<Typography className={classes.heroText}>Converse with anyone with any language</Typography>
+						</Hidden>
+					</Hidden>
+				</Box>
+			</Grid>
```

Comment:
There is a lot of repeated code in these two components, you may want to consider how you can create some reusable components. Here are a couple of ways how:
- Create a component that wraps all Registration pages - you can use the children props to inject the form into this component 
- Create a reusable component for parts of the page that are the same (like the left side image)